### PR TITLE
jsawk: test added

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/jsawk.rb [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+  sha1 "4f2c962c8a5209764116457682985854400cbf24"
 
-  head 'https://github.com/micha/jsawk.git'
+  head "https://github.com/micha/jsawk.git"
 
-  depends_on 'spidermonkey'
+  depends_on "spidermonkey"
 
   def install
-    mv "README.markdown", "README"
```
Comment (mikemcquaid) [inline Library/Formula/jsawk.rb]: Why this change, out of interest? I think this may be there as `README` files get installed to the `prefix` by default but `README.markdown` might not.

Comment (bfontaine) [inline Library/Formula/jsawk.rb]: Ok, I wasn’t aware of this feature and thought this line was useless.

Comment (xu-cheng) [inline Library/Formula/jsawk.rb]: Just out of curiosity, where's the code relating to install README file in homebrew core? I did some searching, no result.

IMHO, if this is a feature in homebrew, we should install README, README.txt, README.md, README.rst, etc automatically.

Comment (mikemcquaid) [inline Library/Formula/jsawk.rb]: I can't find it but I'm sure it used to be there CC @Homebrew/owners

Comment (xu-cheng) [inline Library/Formula/jsawk.rb]: Ok, I find it. It's [here](https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/metafiles.rb). Should we extend the list?

Comment (mikemcquaid) [inline Library/Formula/jsawk.rb]: nice work. Yeh, extending that list seems to make sense.

Comment (xu-cheng) [inline Library/Formula/jsawk.rb]: Will submit a PR.

Comment (bfontaine) [inline Library/Formula/jsawk.rb]: Btw I fixed the PR @mikemcquaid

Comment (xu-cheng) [inline Library/Formula/jsawk.rb]: Sorry for highjack this PR. But a quick question, do we want to extend the `ext` list. OR just ignore the `ext` at all so `README.whatever` can be installed?

Comment (mikemcquaid) [inline Library/Formula/jsawk.rb]: Extend I think. Wouldn't want e.g. `README.html`

Comment (xu-cheng) [inline Library/Formula/jsawk.rb]: While, `README.html` is in the list now.

Comment (mikemcquaid) [inline Library/Formula/jsawk.rb]: heh, stupid me. Might be worth grepping other formulae and seeing if there's other cases that makes sense. If not, just `.markdown` will work for me.

Comment (xu-cheng) [inline Library/Formula/jsawk.rb]: I think a good standard is what ext files will be picked up by Github to show in the repo's front page. But I haven't found any related document yet.

Comment (bfontaine) [inline Library/Formula/jsawk.rb]: These might be [here (markdown)](https://github.com/github/markup/blob/master/lib/github/markup/markdown.rb#L28), [and here (others)](https://github.com/github/markup/blob/master/lib/github/markups.rb#L6-L29).
